src/Infrastructure/LocalImportFileWriter.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\PimMigration\Infrastructure;
+
+use Akeneo\PimMigration\Domain\ImportFileWriter;
+
+/**
+ * Write a CSV import file locally.
+ *
+ * @author    Laurent Petard <laurent.petard@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ */
+class LocalImportFileWriter implements ImportFileWriter
+{
+    /**
+     * All the data must have the same keys.
+     */
+    public function write(array $data, string $filePath): void
+    {
+        $fileHandle = fopen($filePath, 'w');
```

Comment:
Why don't you use the FileSystemHelper ?

Reply:
Yes I could add the method dto FileSystemHelper instead of adding a service